lib/PuppeteerSharp/ElementHandle.cs

```diff
@@ -61,7 +61,7 @@ public async Task ScreenshotAsync(string file, ScreenshotOptions options)
 
             var data = await ScreenshotDataAsync(options).ConfigureAwait(false);
 
-            using (var fs = new FileStream(file, FileMode.Create, FileAccess.Write))
+            using (var fs = new FileStream(file, FileMode.Create, FileAccess.Write, FileShare.Read, 4096, true))
```

Comment:
Can we move all the `4096` to a constant?
It’s a shame that it’s an internal constant I the FileStream class.
I can buy an internal constant in Frame or Page